stripe/src/main/java/com/stripe/android/model/parsers/NextActionDataParser.kt

```diff
@@ -150,6 +152,37 @@ internal class NextActionDataParser : ModelJsonParser<StripeIntent.NextActionDat
         }
     }
 
+    internal class WechatPayRedirectParser :
```

Comment:
Can we reuse the existing `WeChat` parser?

Reply:
the field names are different, `WeChat` parser is looking for names like "android_appId" but `WechatPayRedirectParser` for "app_id"